docs/testplan/srv6/SRv6-phoenixwing-ptf-testplan.md

```diff
@@ -0,0 +1,271 @@
+ cookie=0x0, duration=7718.202s, table=0, n_packets=0, n_bytes=0, priority=10,tcp,in_port="VM0104-t3",tp_src=179 actions=output:"VM0102-t1",output:"inje-vms9-1-39"
+ cookie=0x0, duration=7718.168s, table=0, n_packets=0, n_bytes=0, priority=10,tcp,in_port="VM0104-t3",tp_src=22 actions=output:"VM0102-t1",output:"inje-vms9-1-39"
+ cookie=0x0, duration=7718.158s, table=0, n_packets=264, n_bytes=25384, priority=10,tcp6,in_port="VM0104-t3",tp_src=179 actions=output:"VM0102-t1",output:"inje-vms9-1-39"
+ 
+ ...
+```
+### Setup configuration files
+Instead of using mininet xml, we want to use configdb json files to apply initial underlay configuraiton to each device. dut_cfg_file_loc is used in the topology file to provide the path for DUT's configdb json file and use cfg_file_loc under each VM to provide each VM's configdb json file. 
+
+```
+dut_cfg_file_loc : ../tests/srv6/7nodes_cisco/P1.json
+sonic_password_set : "admin"
+
+configuration:
+  PE1:
+    cfg_file_loc : ../tests/srv6/7nodes_cisco/PE1.json
+    hwsku: cisco-8101-p4-32x100-vs
+```
+
+This test case's configuration files are stored at tests/srv6/7nodes_cisco/
+
+### Launch 7-node PTF Testbed
+We follow existing work flow to launch this testbed in three steps. P1 is the DUT from existing pytest infrastructure's point of view.
+
+#### Launch six helper VMs
+Use the exsiting testbed-cli.sh's option start-topo-vms to start helper VMs based on the defined topology file. The sample launching command is the following.
+
+```
+cd /data/sonic-mgmt/ansible;  export PACKAGE_INSTALLATION=false; ./testbed-cli.sh -t vtestbed.csv -m veos_vtb  -k vsonic  start-topo-vms vms-kvm-ciscovs-7nodes password.txt'
+```
+
+Note: PACKAGE_INSTALLATION is set to let phytest skip package installation. The default behavior is always to install packages which include docker. This docker installation causes the problem since the command needs to be executed inside docker sonic-mgmt-test.
+
+#### Start DUT VM and bind all VMs based on the defined topology file
+Use the exsiting testbed-cli.sh's option add-topo to start DUT VM, a.k.a P1 in this topology. This step would also set up needed links among 7 VMs and apply configurations to previous launched helper VMs
+```
+cd /data/sonic-mgmt/ansible;  export PACKAGE_INSTALLATION=false; ./testbed-cli.sh -t vtestbed.csv -m veos_vtb  -k vsonic  add-topo vms-kvm-ciscovs-7nodes password.txt
+```
+#### Apply DUT's configuration
+Use the exsiting testbed-cli.sh's option deploy-mg to apply configurations to DUT, 
+```
+cd /data/sonic-mgmt/ansible;  export PACKAGE_INSTALLATION=false; ./testbed-cli.sh -t vtestbed.csv -m veos_vtb  -k vsonic  deploy-mg vms-kvm-ciscovs-7nodes veos_vtb password.txt'
+```
+#### Run Test Cases
+We use the following command to run test cases on this 7-node testbed. Some options used are highlighted below
+
+* "-c" is used to specify test cases.
+* "-u" is used to skip pretest and posttest, since we don't use mininet configurations.
```

Comment:
What is `mininet configuration`?

Reply:
Current topo_t0's DUT configurations is generated from mingraph configuration which is generated from topology yml. 

Since we don't use minigraph in this configuration, we need to skip pretest and post test.

Reply:
i change it to minigraph configuration to be more precise.

Reply:
fixed